docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.info.yml

```diff
@@ -3,14 +3,20 @@ type: theme
 base theme: gin
 description: "Gin specific overrides unique to Stanford Humsci"
 core_version_requirement: ^8.8 || ^9 || ^10
-version: '1.0'
+version: "1.0"
 hidden: false
+dependencies:
+  - humsci_basic:humsci_basic
+  - hs_layouts:hs_layouts
 
 libraries:
   - claro/global-styling
   - gin/gin
   - su_humsci_gin_admin/global-styling
   - su_humsci_gin_admin/fix-autocomplete-width
+  - su_humsci_gin_admin/humsci-traditional-styling
+  - su_humsci_gin_admin/humsci-colorful-styling
```

Comment:
```suggestion
```

I'd drop these and include them programmatically since they're conditional includes. See other comments.

Reply:
Done